charts/opensearch/values.yaml

```diff
@@ -306,7 +306,7 @@ securityConfig:
     # * If you define securityConfigSecret, the chart will assume this secret is
     #   created externally and mount it.
     # * It is an error to define both data and securityConfigSecret.
-    securityConfigSecret: ""
+    securityConfigSecret: "security-configs-secret"
```

Comment:
As per the comment directly above this line this will break as soon as you define `data` fields.

https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/templates/statefulset.yaml#L143-L145